peerconnection_media_test.go

```diff
@@ -1074,3 +1074,99 @@ func TestPlanBMultiTrack(t *testing.T) {
 	assert.NoError(t, pcOffer.Close())
 	assert.NoError(t, pcAnswer.Close())
 }
+
+// TestPeerConnection_Start_Right_Receiver tests that the right
+// receiver (the receiver which transceiver has the same media section as the track)
+// is started for the specified track
+func TestPeerConnection_Start_Right_Receiver(t *testing.T) {
+	isTransceiverReceiverStarted := func(pc *PeerConnection, mid string) (bool, error) {
+		for _, transceiver := range pc.GetTransceivers() {
+			if transceiver.Mid() != mid {
+				continue
+			}
+			if transceiver.Receiver() == nil {
+				return false, nil
+			}
+			select {
+			case <-transceiver.Receiver().received:
+				return true, nil
+			default:
+				return false, nil
```

Comment:
I think you can just use `return transceiver.Receiver() != nil && transceiver.Receiver().haveReceived()` instead of all of these lines.

Perhaps this could be changed to `hasReceived` to keep it consistent with `rtpsender.go` which has a `hasSent` function.

Reply:
@jeremija Done!